src/controllers/likings.controller.js

```diff
@@ -10,6 +10,15 @@ const LikingsController = {
     } catch (error) {
       return sendResult(res, 400, 'Wrong accommodation ID sent');
     }
+  },
+
+  async getAccommdationLikes(req, res) {
+    try {
+      const { accommodationId } = req.params;
+      await LikingService.getLikes({ accommodationId }, res);
+    } catch (error) {
+      return sendResult(res, 400, 'Wrong accommodation ID sent');
```

Comment:
great job but I think to try-catch isn't required for this get method just try to test all edge cases otherwise goo job @alexismajya